mmdet3d/datasets/sunrgbd_dataset.py

```diff
@@ -60,6 +60,43 @@ def __init__(self,
             filter_empty_gt=filter_empty_gt,
             test_mode=test_mode)
 
+    def get_data_info(self, index):
+        """Get data info according to the given index.
+
+        Args:
+            index (int): Index of the sample data to get.
+
+        Returns:
+            dict: Data information that will be passed to the data \
+                preprocessing pipelines. It includes the following keys:
+
+                - sample_idx (str): Sample index.
+                - pts_filename (str): Filename of point clouds.
+                - file_name (str): Filename of point clouds.
+                - ann_info (dict): Annotation info.
+        """
+        info = self.data_infos[index]
+        assert info['point_cloud']['lidar_idx'] == info['image']['image_idx']
+        sample_idx = info['point_cloud']['lidar_idx']
+        pts_filename = osp.join(self.data_root, info['pts_path'])
+        img_filename = osp.join(self.data_root, info['image']['image_path'])
+        calib = info['calib']
+
+        input_dict = dict(
```

Comment:
We should support to decide which modality data to be load according to `self.modality`.

Reply:
By the way, I noticed that self.modality in kitti_dataset.py isn't used